queries.sql

```diff
@@ -7,4 +7,11 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered,SUM(escape_attempts) FROM animals GROUP BY neutered ORDER BY sum DESC;
+SELECT species, MIN(weight_kg),MAX(weight_kg) FROM animals  GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals  WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
```

Comment:
- [ ] I think it would be better if you update this query because right now, it excludes all animals born on `1990-01-01` and `2000-01-01`.
I suggest you use this approach to fix the issue:
```SQL
....WHERE date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12'
```